warehouse/migrations/versions/778f1c01a019_release_add_provides_extra_and_dynamic_.py

```diff
@@ -0,0 +1,129 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+release: add provides_extra and dynamic columns
+
+Revision ID: 778f1c01a019
+Revises: 4d1b4fcc4076
+Create Date: 2023-10-12 13:04:25.039108
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "778f1c01a019"
+down_revision = "4d1b4fcc4076"
```

Comment:
```suggestion
down_revision = "81f9f9a60270"
```
We've had a migration merged in the meantime.